setup.py

```diff
@@ -346,7 +346,7 @@ def run_setup(with_extensions):
         ext_modules=ext_modules,
         cmdclass=cmdclass,
         package_dir={"": "."},
-        python_requires=">=3.6, <4",
+        python_requires=">=3.7, <4",
```

Comment:
This change is likely debatable since we're not explicitly breaking compatibility.

Reply:
Eh I think it is ok to drop. We are not testing it any more. So we really can't guarantee things work there.